### ISSUE TITLE
TOPDON TC001 Plus

### ISSUE THREAD

**Issue by mikelmujika11** [leswright1977/PyThermalCamera#19]
Hi,

I noticed that TOPDON has released a new camera, TC001 Plus, which in addition to the thermographic camera, includes a visual camera that allows edge detection. The access and processing of the information from this camera would be similar to the one shown in this repo?

Thanks,

**Comment by LiqunXu** [comment]
I have the same question.

**Comment by 92es** [comment]
To [en/dis]able the "Fusion Mode" feature would most likely require sending commands to the camera versus just reading from the camera.

![Topdon TC001 PLUS Fusion Mode](https://github.com/leswright1977/PyThermalCamera/assets/76127081/0ffd3ea1-be4d-4a0a-9368-84efea5cc4bd)

**Comment by Erudition** [comment]
No, I'm pretty sure you just run edge-detect on the normal camera's output, then overlay it on the thermal output.
